pkgs/applications/networking/powerdns-admin/default.nix

```diff
@@ -125,6 +125,6 @@ in stdenv.mkDerivation rec {
     description = "A PowerDNS web interface with advanced features";
     homepage = "https://github.com/ngoduykhanh/PowerDNS-Admin";
     license = licenses.mit;
-    maintainers = with maintainers; [ zhaofengli ];
+    maintainers = with maintainers; [ flakebi zhaofengli ];
```

Comment:
This probably needs to be a capital **F**lakebi (I guess nix is case-sensitive?)

Reply:
Oops, thanks!